src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -26,6 +26,12 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
 
+        <!-- Json Normalizer -->
+        <service id="serializer.normalizer.json" class="Symfony\Component\Serializer\Normalizer\JsonSerializableNormalizer" public="false">
```

Comment:
I would say that we should handle it in PHP code instead of using Composer rules here (@dunglas did a similar approach for new normalizers).